admin-dev/themes/new-theme/js/app/pages/translations/components/principal/index.vue

```diff
@@ -129,9 +161,11 @@
         return this.$store.state.currentDomain;
       },
       currentDomainTotalTranslations() {
+        /* eslint-disable max-len */
```

Comment:
Can be removed by a eol before `.replace`

Reply:
maybe later :D